unified-codes/tools/scripts/dgraph/migration/v1/import.js

```diff
@@ -0,0 +1,161 @@
+const fs = require('fs').promises;
+const excel = require('exceljs');
+const jsonFormat = require('json-format');
+
+const inputFilename = process.argv[2] || './products.xlsx';
+const outputFilename = process.argv[3] || './import.json';
+const items = [];
+
+const properties = [
+  'code',
+  'title',
+  'type',
+  'ddd',
+  'rxnav',
+  'eml',
+  'atc',
+  'unspsc',
+  'nzulm',
+  'form_category',
+  'form',
+  'code',
+  'title',
+  'strength',
+  'ddd',
+  'rxnav',
+  'eml',
+  'atc',
+  'unspsc',
+  'nzulm',
+];
+const addPropertiesToProduct = (product, colNumber, value) => {
+  const property = properties[colNumber - 1];
+  product[property] = value;
+};
```

Comment:
Hmm... I think this is only used in `processRow`, could maybe be a nested function?